test/e2e/common_test.go

```diff
@@ -413,6 +413,10 @@ func (p *PodmanTestIntegration) BuildImage(dockerfile, imageName string, layers
 // PodmanPID execs podman and returns its PID
 func (p *PodmanTestIntegration) PodmanPID(args []string) (*PodmanSessionIntegration, int) {
 	podmanOptions := p.MakeOptions(args, false)
+	if !p.RemoteTest {
+		cacheOptions := []string{"--storage-opt", fmt.Sprintf("%s.imagestore=%s", p.ImageCacheFS, p.ImageCacheDir)}
+		podmanOptions = append(cacheOptions, podmanOptions...)
+	}
```

Comment:
Not so keen on duplicating this code, since it makes any related future improvements/fixes more error prone.  IMHO better to keep it in the purpose-built `MakeOptions()` flavor of functions.  But I'm also not the expert.  Attempting to cherry-pick your other changes in this PR over to #4118 to see if perhaps that's more palatable...

Reply:
Alternatively we can merge 4118 and I can remove this change

Reply:
That might simpler, and focus this PR on the SIGPOLL / SIGFPE part of the problem.  I'm running the full suite of tests in 4118 now (before was only running --sig-proxy tests).